components/ActivityCard/index.js

```diff
@@ -1,27 +1,20 @@
 import Image from "next/image";
 import styled from "styled-components";
+import Link from "next/link";
+import StyledImageComponent from "../StyledImage";
 
 export default function ActivityCard({ activity }) {
   return (
     <article>
-      <ImageContainer>
-        <StyledImage src={activity.image} alt={activity.title} fill />
-      </ImageContainer>
-
-      <h2>{activity.title}</h2>
+      <Link href={`/${activity.id}`}>
+        <StyledImageComponent src={activity.image} alt={activity.title} />
+        <h2>{activity.title}</h2>
+      </Link>
       <span>
         {activity.area}, {activity.country}
       </span>
-      <div>{activity.category.join(", ")}</div>
-      <p>{activity.description}</p>
+      <br />
```

Comment:
Doing styling with `<br />`  is not a good practice. Can you imagine a better solution without `<br />` ?

Reply:
Thanks, we tried another solution.